### ISSUE TITLE
Publish the rust sdk to crates.io

### ISSUE THREAD

**Issue by octol** [nymtech/nym#2856]

**Comment by sneurlax** [comment]
Yes, soon I will proceed to publish mirror crates in order to be able to publish crates which use nym-sdk.  I would much prefer to use officially-published crates
